packages/flutter_test/lib/src/controller.dart

```diff
@@ -400,6 +400,19 @@ abstract class WidgetController {
     });
   }
 
+  /// A simulator of how the framework handles a serials of [PointerEvent]s
+  /// received from the flutter engine.
+  ///
+  /// The [PointerEventPacket.timeStamp] is used for time delay of an event
+  /// packet relative to the starting point of the method call.
+  ///
+  /// Returns a list of the difference between [PointerEventPacket.timeStamp]
+  /// and the real timestamp when the event packet is processed. The closer
+  /// these values are to zero the more faithful it is to the `records`.
```

Comment:
I'm not sure what this paragraph means. Shouldn't this always be zero in a unit test, since the clock is artificial?

Reply:
For unit test yes. But the whole idea for this is to make the same code and API used for unit test as well as integration test on a device. 
E2E for example is using WidgetTester for running tests on device.